### PR TITLE
Simpler version of Not Visible.

### PR DESCRIPTION
**Pull request by stbof**
DOC-1045
DOC-1046

Replaces https://github.com/cockroachdb/docs/pull/12729

### CODE REVIEW

**Review by ajwerner — APPROVED** [review]
:lgtm:

> Reviewed 2 of 4 files at r1, 4 of 4 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14871#-:-N9jHKVkDHmQ9WN5MaqG:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @stbof)

File: v22.2/alter-column.md [hunk 1]
```diff
@@ -291,6 +292,75 @@ NOTICE: ALTER COLUMN TYPE changes are finalized asynchronously; further schema c
 (10 rows)
 ~~~
 
+### Set the visibility of a column
+
+To specify that a column won't be returned when using `*` in a [`SELECT` clause](select-clause.html), set the `NOT VISIBLE` property. You can set the `NOT VISIBLE` property only on individual columns.
```
Comment (taroface) [inline v22.2/alter-column.md:297]:
```suggestion
To specify that a column will not be returned when using `*` in a [`SELECT` clause](select-clause.html), set the `NOT VISIBLE` property. You can set the `NOT VISIBLE` property only on individual columns.
```

File: v22.2/alter-column.md [hunk 2]
```diff
@@ -291,6 +292,75 @@ NOTICE: ALTER COLUMN TYPE changes are finalized asynchronously; further schema c
...
+
+For example, the `users` table of the [`movr` database](movr.html) contains the `credit_card` column. If you don't want users to see that column when running `SELECT * FROM users;`, you can hide it as follows:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> ALTER TABLE users ALTER COLUMN credit_card SET NOT VISIBLE;
+~~~
+
+When you run `SELECT *`, the column doesn't appear:
```
Comment (taroface) [inline v22.2/alter-column.md:306]:
```suggestion
When you run `SELECT *`, the column does not appear:
```

File: v22.2/create-table.md [hunk 3]
```diff
@@ -166,6 +167,10 @@ Note the following limitations of identity columns:
 
 For an example of an identity column, see [Create a table with an identity column](#create-a-table-with-an-identity-column).
 
+### `NOT VISIBLE` property
+
+The `NOT VISIBLE` property specifies a column won't be returned when using `*` in a [`SELECT` clause](select-clause.html). You can apply the `NOT VISIBLE` property only to individual columns. For an example, see [Show the `CREATE TABLE` statement for a table with a hidden column](show-create.html#show-the-create-table-statement-for-a-table-with-a-hidden-column).
```
Comment (taroface) [inline v22.2/create-table.md:172]:
```suggestion
The `NOT VISIBLE` property specifies a column will not be returned when using `*` in a [`SELECT` clause](select-clause.html). You can apply the `NOT VISIBLE` property only to individual columns. For an example, refer to [Show the `CREATE TABLE` statement for a table with a hidden column](show-create.html#show-the-create-table-statement-for-a-table-with-a-hidden-column).
```

**Review by taroface — APPROVED** [review]
LGTM, just had some suggestions on contractions.

**Pull request merged**